aspnet/getting-started/installing-on-linux.rst

```diff
@@ -3,12 +3,16 @@ Installing ASP.NET 5 On Linux
 
 By `Daniel Roth`_
 
-ASP.NET 5 runs on the .NET Execution Environment (DNX), which is available on multiple platforms, including Linux. This article describes how to install DNX, and therefore ASP.NET 5, on Linux using .NET Core and Mono.
+.. contents:: In this article
+  :local:
+  :depth: 1
 
-In this article:
-  - `Installing on Debian, Ubuntu and derivatives`_
-  - `Installing on CentOS, Fedora and derivatives`_
-  - `Using Docker`_
+Install using prebuild binaries
+-------------------------------
+
+Prebuild binaries for ASP.NET 5 are available (`.tar.gz <http://go.microsoft.com/fwlink/?LinkId=703941>`__) and can be installed as appropriate based on your system configuration.
```

Comment:
The pre-built binaries only include `dnu` and `dnx`. You will still need to install the dependencies `libunwind8 gettext libssl-dev libcurl4-openssl-dev zlib1g libicu-dev uuid-dev ` and build `libuv`